docs/Modules/multichain.mdx

```diff
@@ -1,22 +1,153 @@
 
-## Installation
+### What is the Multichain Validation Module? 🤔
+
+This module allows **Externally Owned Accounts (EOAs)** to authorize and sign user operations (UserOps) for Biconomy Smart Accounts on Multiple chains with single ECDSA signature. 
+
+`Benefits`
+- significantly improves UX for deploying and setting up Smart Accounts on several chains 
+- significantly reduces user friction for multichain operations.
+
+:::tip
+You can use also this module as the **default module** just like ECDSA Validation Module as it has all it's capabilities for sending ops on particular chain.
+:::
+
+#### Key Functions 🔐
+
+- **ECDSA Signature Scheme**: Utilizes the ECDSA secp256r1 curve for secure signing.
+- **Flexible Signer Authentication**: Supports various signer solutions like Privy, Fireblocks, Arcana Auth, Capsule, Lit protocol, Turnkey, Web3Auth, Particle, Magic, Portal etc
+- **EIP-1271 Compliance**: Allowing Smart Accounts to sign Ethereum messages for logging into dApps.
+- **Sign once, Execute on multiple chains**
+
+#### Use Cases 🌟
+
+Enables use cases which require several actions to be authorized for several chains with just one signature required from user.
+
+1. **Multichain intent for example exit all my native tokens for usdc
+2. **Allows a dApp to require just one signature from their user to configure & deploy smart accounts on multiple chains and delegate certain actions with permission via session keys on all those chains
+3. **Cross chain state sync actions e.g transfer ownership of SA, change ens, similar actions on protocol that is deployed on multiple chains
+4. **Onboard user on x chains by deploying their smart account + some action like register in raffle 
+
+### How does it work? 🤔
+
+1. We take all the userOps which we need to sign.
+2. We get userOpHash from those userOps.
+3. Every userOpHash is now a leaf of a Merkle Tree.
+4. Instead of signing several userOpHash separately, user signs one Merkle Root.
+5. Now we can reliably prove on-chain that a userOp is a leaf of a tree whose root has been signed by an authorized user.
+
+Now user can authorize any amount of userOps with just one signature over the Merkle Root of those userOps
+
+
+## Smart Contract Deep Dive 🛠️
+
+You can send a multi-chain signature to send different userops on different chains and be validated by this module. if plain ecdsa signature(65 bytes) is sent then the module assumes it’s form as standard ECDSA validation module
+
+#### User Operation Validation (`validateUserOp`) 🛡️
+
+```solidity
+
+function validateUserOp(
+        UserOperation calldata userOp,
+        bytes32 userOpHash
+    ) external view virtual override returns (uint256) {
+
+       // userOp.signature contains both the signature and the validation module's address
+
+        (bytes memory moduleSignature, ) = abi.decode(
+            userOp.signature,
+            (bytes, address)
+        );
+
+
```

Comment:
can we remove extra new lines and spaces from the code

Reply:
[Solidity Style Guide](https://docs.soliditylang.org/en/latest/style-guide.html#blank-lines)

Reply:
I have copied this from code in scw repo which uses solidity linter